projects/spectator/package.json

```diff
@@ -23,9 +23,9 @@
     "angular testing dumb components"
   ],
   "dependencies": {
-    "@testing-library/dom": "6.1.0",
-    "jquery": "3.5.0",
-    "replace-in-file": "^4.1.3",
+    "@testing-library/dom": "7.26.5",
```

Comment:
Are there any breaking changes in these packages?

Reply:
No breaking changes, `v7.0.1` was built and is already using these versions, and I've installed it without issues so they are compatible